app/backend/src/couchers/interceptors.py

```diff
@@ -123,35 +127,45 @@ def _store_log(self, method, status_code, duration, user_id, request, response,
                     user_id=user_id,
                     request=req_bytes,
                     response=res_bytes,
+                    details=grpc_details,
                     traceback=traceback,
                 )
             )
-        logger.debug(f"{user_id=}, {method=}, {duration=} ms")
+        logger.debug(f"{user_id=}, {method=}, {grpc_details=}, {duration=} ms")
 
     def intercept_service(self, continuation, handler_call_details):
         handler = continuation(handler_call_details)
         prev_func = handler.unary_unary
         method = handler_call_details.method
 
         def tracing_function(request, context):
+            user_id = getattr(context, "user_id", None)
```

Comment:
This needs to happen *after* the call; as the auth interceptor is after this one... `context` is mutated in `prev_func`.